qtgui/dockbookmarks.cpp

```diff
@@ -151,7 +153,7 @@ void DockBookmarks::on_tableWidgetTagList_itemChanged(QTableWidgetItem *item)
     if(ui->tableWidgetTagList->m_bUpdating) return;
 
     int col = item->column();
-    if(!col==1) return;
+    if(!(col==1)) return;
```

Comment:
I have to check because I am not familiar with this part of the code. I would have written col != 1 so it could be intentional.